listenbrainz/webserver/views/user_timeline_event_api.py

```diff
@@ -1049,6 +1175,55 @@ def get_personal_recording_recommendation_events(
                 hidden=False,
             ))
         except pydantic.ValidationError as e:
-            current_app.logger.error('Validation error: ' + str(e), exc_info=True)
+            current_app.logger.error(
+                'Validation error: ' + str(e), exc_info=True)
+            continue
+    return events
+
+
+def get_thanks_events(
+    user: dict,
+    min_ts: int,
+    max_ts: int,
+    count: int
+) -> List[APITimelineEvent]:
+    """ Gets all thanks events in the feed.
+    """
+
+    thanks_events_db = db_user_timeline_event.get_thanks_events_for_feed(
+        db_conn,
+        user_id=user['id'],
+        min_ts=min_ts,
+        max_ts=max_ts,
+        count=count,
+    )
+
+    events = []
+    for event in thanks_events_db:
+        try:
+            thank = APIThanksEvent(
+                user_id=event.user_id,
+                created=event.created.timestamp(),
+                original_event_id=event.metadata.original_event_id,
+                original_event_type=event.metadata.original_event_type.value,
+                blurb_content=event.metadata.blurb_content,
+                thanker_id=event.metadata.thanker_id,
+                thanker_username=event.metadata.thanker_username,
+                thankee_id=event.metadata.thanker_id,
```

Comment:
This looks incorrect:
```suggestion
                thankee_id=event.metadata.thankee_id,
```

Reply:
Thanks for pointing this out!